torchdp/compute_dp-sdg_privacy.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+"""
+Both `apply_dp_sgd_analysis()` and `get_privacy_spent()` functions
+ are based on Google's TF Privacy:
+ https://github.com/tensorflow/privacy/blob/master/tensorflow_privacy/privacy/analysis/compute_dp_sgd_privacy.py
+
+Command-line script for computing privacy of a model trained with DP-SGD.
+The script applies the RDP accountant to estimate privacy budget of an iterated
+ Sampled Gaussian Mechanism.
+
+Example:
+  compute_dp_sgd_privacy
+    --dataset-size=60000 \
+    --batch-size=256 \
+    --sigma=1.12 \
+    --epochs=60 \
+    --delta=1e-5
+The output states that DP-SGD with these parameters satisfies (2.92, 1e-5)-DP.
+"""
+import argparse
+import math
+from privacy_analysis import compute_rdp, get_privacy_spent
+
+
+def apply_dp_sgd_analysis(q, sigma, steps, orders, delta):
```

Comment:
can you add parameter docstring

Reply:
```suggestion
def apply_dp_sgd_analysis(q, sigma, steps, orders, delta):
    """Compute and print results of DP-SGD analysis.

    Args:
        q: the sample rate in SGD.
        sigma: the noise_multiplier in compute_rdp(), ratio of the
               standard deviation of the Gaussian noise to
               the l2-sensitivity of the function to which it is added
        steps: the number of steps.
        orders: an array (or a scalar) of RDP orders.
  """
```

Reply:
In addition, note that previous docstring was refering a TF-specific file and yet wasn't available in pytroch-dp context.